tests/network/vmi_multus.go

```diff
@@ -1154,13 +1134,7 @@ var _ = Describe("[Serial]SRIOV", func() {
 					// It may take some time for the VMI interface status to be updated with the information reported by
 					// the guest-agent.
 					Eventually(func() error {
-						updatedVMI, err := virtClient.VirtualMachineInstance(vmi.Namespace).Get(vmi.Name, &metav1.GetOptions{})
-						Expect(err).NotTo(HaveOccurred())
-						interfaceName, err := getInterfaceNameByMAC(updatedVMI, mac)
-						if err != nil {
-							return err
-						}
-						return checkMacAddress(updatedVMI, interfaceName, mac)
+						return checkMacAddress(vmi, findInterfaceHavingMAC(virtClient, vmi, mac), mac)
```

Comment:
Is this intended? Don't you want to remove the `Eventually` or the `Poll`?

Reply:
Good catch. It can now be an `Expect` .

Reply:
Done.